pkgs/applications/misc/blender/versions.json

```diff
@@ -0,0 +1,28 @@
+{
+
+    "testing": {
```

Comment:
Forgot to remove, will do so later. Planning on adding stable versions only in this json.